pkg/virtctl/imageupload/imageupload.go

```diff
@@ -926,3 +935,34 @@ func handleEventErrors(client kubecli.KubevirtClient, pvcName, dvName, namespace
 
 	return nil
 }
+
+func createDataSource(client kubecli.KubevirtClient, name, namespace string) error {
+	if dv, _ := client.CdiClient().CdiV1beta1().DataSources(namespace).Get(context.Background(), name, metav1.GetOptions{}); dv != nil {
+		fmt.Printf("DataVolume %s/%s is already exists\n", namespace, name)
```

Comment:
`DataSource` but if it already exists we still need to patch it to point to the new `PVC`.

Reply:
Thanks Lee :)